windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -109,10 +111,17 @@ mkdir $NSIS_UNZU_DIR
 $ProgressPreference = "silentlyContinue"
 # Handle SSL correctly.
 [Net.ServicePointManager]::SecurityProtocol = 'TLS12'
-# Pretend to be curl for Sourceforge redirects to work.
-Invoke-WebRequest "$RUBY_INSTALLER_LINK" -OutFile "$RUBY_INSTALLER" -UserAgent "curl/7.60.0"
-Invoke-WebRequest "$NSIS_INSTALLER_LINK" -OutFile "$NSIS_INSTALLER" -UserAgent "curl/7.60.0"
-Invoke-WebRequest "$NSIS_UNZU_INSTALLER_LINK" -OutFile "$NSIS_UNZU_ZIP" -UserAgent "curl/7.60.0"
+
+# Need help with Chocolatey? See https://chocolatey.org/install
```

Comment:
What makes you think it's incomplete?
The reason is in the comment: if anyone needs help because that step fails, they should look at that page. I don't have the ability to test all the possible environments where this could run.

Reply:
Removed code.